### DISCUSSION TITLE
Uninstall

### DISCUSSION THREAD

**Discussion by Zweaky** [filamentphp/filament#1389] [Help] [answered]
How do I uninstall filament from my laravel project?

**Comment by ryangjchandler** [comment, marked as answer]
Remove the Composer package and remove the app/Filament folder. 

If you published the config, you can also remove that too.

**Reply by Zweaky** [reply]
where would I find the composer package and when I do find it do I just delete or is there some pruning to be done.

**Reply by MACscr** [reply]
> where would I find the composer package and when I do find it do I just delete or is there some pruning to be done.

Same way you installed it, but instead of a require, you would simply do a remove. Then remove the files above as ryan pointed out.

**Reply by Zweaky** [reply]
Thanks!

**Reply by ubiratan-neto** [reply]
The commands are 'composer remove filament/filament' and then remove the app/Filament folder.
